salt/_modules/metalk8s_volume.py

```diff
@@ -0,0 +1,54 @@
+'''
+Various functions to interact with Salt pillar.
+'''
+import re
+
+import logging
+
+import salt.utils.json
+from salt.exceptions import CommandExecutionError
+
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'metalk8s_volume'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+# check if the storage class returns information about itself
+# should be in a proper state
+def get_storageclass_validity(storageclassname, **kwargs):
+    storageclass_info = __salt__['metalk8s_kubernetes.read_storage_class'](
+        storageclassname, **kwargs
+    )
+    if storageclass_info is None:
+        return None
+    return storageclass_info
+
+
+# simple helper to convert kubernetes size object to human readable form
+# e.g 43Gi from Kubernetes yields 46170898432
+def to_bytes_converter(quantity):
```

Comment:
Can't find anything in `python-kubernetes`.

Reply:
Not in the stdlib I'm affraid.

As to bring an external dependency for a 10 lines (counting code only) function, I'm not convinced by the JS way of life…

That being said, would be nice to have some unit tests for small helpers like that.
Probably out of scope of this PR, but we could have some doctest at least in the docstring at least (instead of having a simple comment).